.github/workflows/gating.yaml

```diff
@@ -99,3 +99,41 @@ jobs:
           # * DL3041 - Specify version with dnf install -y <package>-<version>
           ignore: DL3041
           failure-threshold: warning
+
+  image-build:
+    name: Container Image Build
+    needs: hadolint
+    runs-on: ubuntu-latest
```

Comment:
Hi @hluk 
Could I ask if all the github ci runs on the ubuntu-latest?

Reply:
AFAIK, most of our workflows use ubuntu-latest GitHub-hosted runners (or a specific/fixed version of ubuntu - e.g. ubuntu-20.04).